test/ui/pages/config/ContentPageContaier.test.js

```diff
@@ -0,0 +1,69 @@
+import { mapStateToProps, mapDispatchToProps } from 'ui/pages/config/ContentPagesContainer';
+import { HOMEPAGE_PAYLOAD } from 'test/mocks/pages';
+
+jest.mock('state/pages/selectors', () => ({
```

Comment:
Don't repeat the mocks inside the test files. Please rewrite like this to use the mock:

```javascript
import { getPageTreePages } from 'state/pages/selectors';

jest.mock('state/pages/selectors', () => ({
  getPageTreePages: jest.fn(),
}));

getPageTreePages.mockReturnValue(HOMEPAGE_PAYLOAD);
```

Reply:
Also, this selector is intended to return an array, why are you mocking returning a single object?